cypress/e2e/signIn.cy.js

```diff
@@ -1,11 +1,79 @@
+/* eslint-disable max-len */
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
   beforeEach(() => {
+    cy.visit('/login');
+  });
+
+  it('should provide the ability to login with registered data', () => {
+    const username = 'tomsmith';
+    const password = 'SuperSecretPassword!';
```

Comment:
```suggestion
    const username = 'tomsmith';
    const password = 'SuperSecretPassword!';
```
you use these variables a lot of times, so move them to the describe section and reuse them in every place you need. Remove extra variables